### ISSUE TITLE
FYI: info for Mac hosts in scale_2018_usb.pdf Lab 1

### ISSUE THREAD

**Issue by toddzebert** [e-ale/Slides#1]
the host device is not /dev/ttyACM{n} but rather /dev/tty.usbmodem{n} where {n} here is a bunch of numbers

**Comment by robbawebba** [comment]
@toddzebert Happy to hear someone else was having trouble with this lab on a mac! I was trying the exercise with /dev/tty.usbmodem{n} but it unfortunately didn't work for me.

**Comment by toddzebert** [comment]
@robbawebba you have both USB cables connected? do you not see "usbmodem" when you do `ls /dev/{tty,cu}.*` on your Mac?

**Comment by robbawebba** [comment]
@toddzebert yup! I see the usbmodem device, but I was unable to communicate with /dev/ttyGS0 on the PocketBeagle through the usbmodem device. I will try again later tonight on a linux machine. Once I get it running there, I'll probably try again on a Mac and let you know how it goes.

**Comment by toddzebert** [comment]
Check your tty's on the device as mine was `/dev/ttyGS1`
